Sources/ThirdParty/JSONWebToken/JSONWebToken.swift

```diff
@@ -146,8 +146,11 @@ public struct JSONWebToken {
     public let payload : Payload
     let base64Parts : (header : String,payload : String, signature : String)
     
-    public init(string input: String) throws {
 
+    public init(string input: String) throws {
+        try self.init(string: input, typeHeader: "JWT")
+    }
+    public init(string input: String, typeHeader : String?) throws {
```

Comment:
Such constructor can cause unpredictable behavior as I can call it as: `init(input: input, typeHeader: **nil**)`
I suggest to redo to:
```
public init(string input: String, typeHeader : String = "JWT") throws
```